doc/source/whatsnew/v2.1.0.rst

```diff
@@ -295,6 +295,7 @@ Other API changes
 
 Deprecations
 ~~~~~~~~~~~~
+- Deprecated the behavior of :meth:`Series.argsort` in the presence of NA values; in a future version these will be sorted at the end instead of giving -1 (:issue:`??`)
```

Comment:
```suggestion
- Deprecated the behavior of :meth:`Series.argsort` in the presence of NA values; in a future version these will be sorted at the end instead of giving -1 (:issue:`54219`)
```